MyArrayList.java

```diff
@@ -1 +1,104 @@
+package com.company;
+
+public class MyArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_CAPACITY = 10;
+    private int position = 0;
+    private Object[] array = new Object[DEFAULT_CAPACITY];
+    private StringBuilder stringBuilder = new StringBuilder();
+    private int size = array.length;
+
+    public void printArrayList() {
+        System.out.println("My array list: ");
+        stringBuilder.setLength(0);
+        for (int j = 0; j < size; j++) {
+            stringBuilder.append(array[j]).append(", ");
+        }
+        System.out.println(stringBuilder);
+    }
+
+    @Override
+    public void add(T value) {
+        check(position);
+        add(value, position);
+        position++;
+    }
+
+    private void check(int index) {
+        if (index < 0 || index > position) {
```

Comment:
try change logic of this method:
if we have wrong index we will increase array length?

Reply:
yes, i corrected it